src/styles/card.css

```diff
@@ -0,0 +1,95 @@
+.card {
+  width: 200px;
+  padding: 0 16px;
+
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__icon {
+  display: block;
+
+  margin: 32px auto 40px;
+  height: 134px;
+  width: 160px;
+}
+
+.card__title {
+  margin-bottom: 4px;
+
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: #060b35;
+  text-align: left;
+}
+
+.card__product {
+  margin-bottom: 16px;
+
+  line-height: 14px;
+  font-size: 10px;
+  font-weight: 400;
+  color: #616070;
+}
+
+.stars__review {
+  padding-left: 18px;
+
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  text-align: center;
+  color: #060b35;
+}
+
+.card__price {
+  display: flex;
+  justify-content: space-between;
+
+  margin-bottom: 16px;
+  line-height: 18px;
+}
+
+.card__word {
+  font-weight: 400;
+  font-size: 12px;
+  color: #616070;
+}
+
+.card__digits {
+  font-weight: 700;
+  font-size: 16px;
+  color: #060b35;
+}
+
+.card__link {
+  display: flex;
+  justify-content: center;
+  margin-bottom: 16px;
+}
+
+.card__item {
+  align-items: center;
+  justify-content: center;
+
+  text-transform: uppercase;
+  font-weight: 700;
+  font-size: 14px;
+  line-height: 16px;
+  color: #fff;
+  text-decoration: none;
+
+  padding: 11px 69px 11px 68px;
```

Comment:
😱
what will be if I will need to change the size of the card or button? I need to calc all this data again?

Reply:
Everything stretches automatically. I tried it and it looks good

Reply:
Let's set the height of the button. Also, button's width should be 100% percent of the parent container